app/javascript/lib/imperial2030.test.js

```diff
@@ -143,7 +143,7 @@ describe('imperial2030', () => {
         expect(game.nations.get(Nation2030.RU).treasury).toEqual(9);
       });
 
-      test('nations cannot be paid less than 0 if they control many units', () => {
+      test('nations lose money if they control many units', () => {
         const game = newGame();
         // Arbitrarily give RU 5 treasury; we want this to increase by 1
```

Comment:
If this comment is incorrect, please fix it.

Reply:
Good catch didn't notice it, will update

Reply:
[67de8dc](https://github.com/Thrillberg/imperial/pull/689/commits/67de8dc92e10c3a6a3c9ecdd3638707aa7953619)